.github/workflows/publish-site.yml

```diff
@@ -13,23 +13,41 @@ jobs:
       - name: Install Dependencies
         run: |
           sudo apt-get update -y
-          sudo apt-get install --fix-missing -y lua-ldoc lua-markdown jq p7zip-full libsdl2-2.0-0 libopenal1
+          sudo apt-get install --fix-missing -y jq p7zip-full libsdl2-2.0-0 libopenal1
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Install Lua Language Server
+        run: |
+          brew install lua-language-server
       - name: Checkout Source Repository
         uses: actions/checkout@v2
-        with:
-          ref: master
       - name: Generate Dynamic Content
         run: |
           bash doc/site/_scripts/get_release_data.sh
           bash doc/site/_scripts/get_engine_data.sh
           git add -f doc/site/_data
-      - name: Run LDoc
+      # This step should be removed when https://github.com/LuaLS/lua-language-server/issues/2977 is resolved.
+      - name: Disable global libaries from Lua config
+        run: |
+          cp doc/site/.luarc.doc.json .luarc.json
```

Comment:
We could move this file to `doc` since it's not site related anyway.

Reply:
Moved into `rts/Lua/library` so it doesn't need to be copied at all.